pyproject.toml

```diff
@@ -215,6 +220,6 @@ allowlist_externals = test/static-code
 setenv =
   NO_QUNIT=1
 commands =
-  pytest: python3 -m pytest -opythonpath= {posargs}
+  pytest: python3 -m pytest -k 'not pcp' -opythonpath= {posargs}
```

Comment:
We have two tox configurations:

* system "tox" -> should run here
* virtualenv (there it wont't work) -> can't run here

Reply:
Resolved!